api/views/search.py

```diff
@@ -0,0 +1,30 @@
+from api import app, db
+from api.models import Story, StoryPOI, POI
+from api.utils import create_response, row_constructor
+from flask import Blueprint, request
+
+SEARCH_URL = "/search"
```

Comment:
since this is a search specifically for pois, i think it would be good to make the endpoint url `/search/pois`